packages/compartment-mapper/src/import-archive-lite.js

```diff
@@ -405,7 +401,7 @@ export const parseArchive = async (
     await pendingJobsPromise;
 
     // eslint-disable-next-line dot-notation
-    return compartment['import'](moduleSpecifier);
+    return compartment['import'](entryModuleSpecifier);
```

Comment:
I assume this is for evasion

Reply:
It was, though that is not necessary any more. It predates some added sophistication to the evasive transforms.